andela/vidar-ah-frontend
# #165484337: Merge landing and welcome page design

#### What does this PR do?
- Fixed landing page

#### Description of task to be completed?
- Improve header to welcome user upon signing in 
- Include search and filter components

#### How should this be manually tested?
- Sign in to the application and observe welcome greeting at the top of the page

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#165484337](https://www.pivotaltracker.com/story/show/165484337)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
